src/styles/blocks/card.scss

```diff
@@ -0,0 +1,159 @@
+.cards {
+  display: grid;
+  grid-template-columns: repeat(1, $card-width);
+  gap: 46px 48px;
+  padding: 50px 40px;
+  place-content: center;
+  place-items: center;
+}
+
+@media (min-width: 488px) {
+  .cards {
+    grid-template-columns: repeat(2, $card-width);
+  }
+}
+
+@media (min-width: 768px) {
+  .cards {
+    grid-template-columns: repeat(3, $card-width);
+  }
+}
+
+@media (min-width: 1024px) {
+  .cards {
+    grid-template-columns: repeat(4, $card-width);
+  }
+}
+
+.product-card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: $card-width;
+  border-radius: $border-radius;
+  border: 1px solid $border-color;
+  margin: 0; /* Remove any default margins */
+  transition: transform 300ms ease; /* Simplified transition */
+
+  /* Rest of the .product-card styles remain the same */
+
+  &__product-image {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 40px;
+  }
+
+  &__product-details {
+    width: $card-content-width-adjustment;
+    margin-inline: $padding-inline;
+    margin-bottom: $margin-bottom;
+
+    &-title {
+      height: 36px;
+      font-size: $font-size-medium;
+      font-weight: $font-weight-medium;
+      line-height: 1.5;
+      text-align: left;
+      color: $secondary-color;
+      margin-top: 0;
+      margin-bottom: 4px;
+    }
+
+    &-description {
+      margin: 0;
+      font-size: $font-size-small;
+      line-height: 14px;
```

Comment:
The `line-height` for `.product-card__product-details-description` is set to `14px`. It is recommended to use unitless values for `line-height` to ensure it scales appropriately with the font size.